src/main.ts

```diff
@@ -16,13 +16,20 @@ const contextPromise = store.dispatch('fetchContext').catch(() => {
   window.location.href = '/login'
 })
 
+const app:Vue = new Vue({ store, router, render: h => h(App) })
```

Comment:
It used to stand here but it is too soon timing issues will happen if we put it here

Reply:
The Vue instance itself is still mounted from the callback. Are there other timing issues then?

Reply:
Router is indeed triggered before fetchContext action is being resolved. Will move this to a separate PR.